submissions/peterIyiola/htmlForms/signuppage.html

```diff
@@ -28,27 +28,36 @@ <h2>Register your Account</h2>
             <p>Already got an account?<a href="#"> sign in</a></p>
          </div>
```

Comment:
@Iyiola-stack You need to remove all the break elements on this page and style with css

Reply:
@Iyiola-stack